lib/widgets/drive/compare.coffee

```diff
@@ -0,0 +1,38 @@
+_ = require('lodash')
+
+containsDeep = (array, item) ->
+	return _.any(_.map(array, _.partial(_.isEqual, item)))
+
+differenceDeep = (x, y) ->
+	return _.filter(x, _.partial(_.negate(containsDeep), y))
+
+createDiffOperation = (type, element) ->
+	return {
+		type: type
+		drive: element
+	}
+
+###*
+# @summary Detect changes regarding drives between different time intervals
+# @function
+# @public
+#
+# @param {Array} - previous drive list
+# @param {Array} - current drive list
+# @returns {Object[]} - current drive list, potential differences with previous one
+#
+# @example
+# compare(previousDrives, currentDrives)
+###
+
```

Comment:
Please remove this blank line. We don't add blank lines between JSDoc annotations and the actual code.